### PR TITLE
Reuse scene and camera objects

### PR DESCRIPTION
**Pull request by BrendanAnnable**
Fixes those pesky memory leaks.

### CODE REVIEW

File: src/client/components/three/three.tsx [hunk 1]
```diff
@@ -54,3 +63,45 @@ export class Three extends Component<{ stage(canvas: Canvas): IComputedValue<Sta
...
+    this.copyToLocalScene(stage.scene)
+    this.copyToLocalCamera(stage.camera)
+    return { camera: this.camera, scene: this.scene }
+  }
+
+  private copyToLocalScene(scene: Scene) {
+    this.scene.copy(scene)
+    this.scene.children.length && this.scene.remove(...this.scene.children)
+    for (const child of scene.children) {
+      // Hack: Remove the parent so that three.js doesn't remove the children from the original scene object.
```
Comment (TrentHouliston) [inline src/client/components/three/three.tsx]: Do we care? Won't the ones using this be recreating the scene object each time anyway?

Comment (BrendanAnnable) [reply src/client/components/three/three.tsx]: Try removing this line and see what happens 😛 (all the stories go blank)

Comment (TrentHouliston) [reply src/client/components/three/three.tsx]: Seems important then! :P

Comment (BrendanAnnable) [reply src/client/components/three/three.tsx]: I agree that it should be regenerated 99% of the time, so I'll still look into why exactly this is needed.

...but I'll merge this for now 🙂

**Pull request merged**